executor/wasm/src/lib.rs

```diff
@@ -62,6 +62,8 @@ const DEFAULT_WASM_ENTRY_POINT: &str = "call";
 
 const DEFAULT_MINT_TRANSFER_GAS_COST: u64 = 1; // NOTE: Require gas while executing and set this to at least 100_000_000 (or use chainspec)
 
+const NAME_FOR_V2_CONTRACT_MAIN_PURSE: &str = "__v2_main_purse";
```

Comment:
I don't think it makes sense to put v2_ into this name.

Reply:
Renamed